Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -932,6 +1011,9 @@ private static InputOverlayDrawableDpad initializeOverlayDpad(Context context,
     // Need to set the image's position
     overlayDrawable.setPosition(drawableX, drawableY);
 
+    // Set opacity
+//    overlayDrawable.setAlpha(overlayOpacity);
```

Comment:
Whats with this commented out piece of code? Should it be enabled or removed?

Reply:
Fixed, removed